sql-project-1/sql-project-1_solution.sql

```diff
@@ -0,0 +1,128 @@
+
+#1 In the ‘dsstudent’ database, create a permanent table named “customer_{your_name}.”
+USE dsstudent;
+
+CREATE TABLE customer_newton
+	(customer_id smallint,
+	name varchar(20),
+	location varchar(20),
+	total_expenditure varchar(20),
+	CONSTRAINT pk_customer_newton PRIMARY KEY (customer_id)
+	);
+
+#2 Insert the following records to the “customer_{your_name}” table:
+
+INSERT INTO customer_newton (customer_id, name, location, total_expenditure)
+VALUES (1701, 'John', 'Newport Beach, CA', '2000'),
+		(1707, 'Tracy', 'Irvine, CA', '1500'),
+		(1711, 'Daniel', 'Newport Beach, CA', '2500'),
+		(1703, 'Ella', 'Santa Ana, CA', '1800'),
+		(1708, 'Mel', 'Orange, CA', '1700'),
+		(1716, 'Steve', 'Irvine, CA', '18000');
+		
+#3 Oops! The value in the field ”total_expenditure” of Steve is not correct. It should be “1800.” Can you update this record?
+UPDATE customer_newton
+SET total_expenditure = '1800'
+WHERE customer_id = 1716;
+
+
+# 4 We would like to update our customer data. Can you insert a new column called “gender” in the “customer_{your_name}” table?
+ALTER TABLE customer_newton
+ADD gender varchar(20);
+
+# 5 Then, update the field “gender” with the following records:
+UPDATE customer_newton

```

Comment:
Can also use this without a WHERE condition to set all to either 'M' or 'F' then update just the ones that aren't that value. Saves a few lines of code.